beta-src/src/webDiplomacyTheme.ts

```diff
@@ -24,9 +51,37 @@ const webDiplomacyTheme = createTheme({
       contrastText: "#000",
     },
   },
-  typography: {
-    fontFamily: "SF Pro Display, Segoe UI, Droid Sans, sans-serif",
-  },
 });
 
+webDiplomacyTheme.typography.body1 = {
+  fontFamily: webDiplomacyTheme.typography.fontFamily,
```

Comment:
MUI makes me set the font-family here...so i just refer to the default font for mui which is roboto.